### ISSUE TITLE
Creature Factory 

### ISSUE THREAD

**Issue by GreatSarmad** [princeton-vl/infinigen#98]
Hi I've been trying to generate scenes with more creatures, herbivore, carnivore, etc. So far managed to get 2 birds only  by changing scene config of snowy mountain, in plains snakes make the spawner cube but it fails to populate it. I would like to see a command that would make it easy for a noob like me. for example"
$BLENDER -noaudio --background --python generate.py -- --seed 0 --task populate {--10 herbivore -- 5 carnivore -- 6 birds} fine_terrain -g plain --input_folder outputs/helloworld/coarse --output_folder outputs/helloworld/fine

from what i understand it\s the configs that manage that but they fail often but pipeline shows populate true.

If anyone in community can show me a proper example as i think I am probably doing something incorrectly. A video tutorial by the creators would be fire too. 
I think overall project is wonderful and look forward to upcoming updates !!!

  
[pipeline_fine.csv](https://github.com/princeton-vl/infinigen/files/12006293/pipeline_fine.csv)

**Comment by araistrick** [comment]
The white cubes are a bug (#86) which I just bandaid patched on `develop` but will fix properly soon. You can crank up the number of creatures by adding commandline gin overrides (IE appending the following to your usual command) `-p compose_scene.max_ground_creatures=7 compose_scene.ground_creatures_chance=1.0` etc

**Comment by GreatSarmad** [comment]
Thanks for the suggestion, but I still get these white blocks hopefully you will fix it in 1.0.2 and I tried videos, i still get them, also tried to force generating ground creature but I see only blocks, I will try to tinker some more, as i really like this project and props for even getting this far! Keep it up! and i am hoping to see the updates from roadmap soon!

**Comment by araistrick** [comment]
Ok, sorry these issues have persisted. v1.0.2 should (hopefully) be out this week but will not feature more changes RE white cubes unless I can find them today/tommorow as a result of this thread. I do have one idea in mind though.

Could you send over the exact command you are running? I must admit I have not thoroughly tested recent changes as theres a lot going on, and knowing exactly what issue to fix would be mildly helpful.

Thanks again for your interest in the project, thrilling to see people digging around / waiting on updates. We will open things up (via better docs, code comments etc) to be more and moree accessible to contributors, and we'd love to have you or anyone contribute as or when you feel its ready / accessible to be worked on. Letting us know what is confusing/buggy is very useful and important.

**Comment by GreatSarmad** [comment]
Sure thing, I hope I am not being silly and mistyping, but here:
for Video : 
python -m tools.manage_datagen_jobs --output_folder outputs/my_videos --num_scenes 10 \
    --pipeline_config local_16GB monocular_video cuda_terrain opengl_gt \
    --cleanup big_files --warmup_sec 1200 --config trailer high_quality_terrain \
    -p compose_scene.max_ground_creatures=7 compose_scene.ground_creatures_chance=1.0
    
For Scenes:
$BLENDER -noaudio --background --python generate.py -- --seed 0 --task coarse -g plain --output_folder outputs/helloworld/coarse
$BLENDER -noaudio --background --python generate.py -- --seed 0 --task populate fine_terrain -g plain --input_folder outputs/helloworld/coarse --output_folder outputs/helloworld/fine
$BLENDER -noaudio --background --python generate.py -- --seed 0 --task render -g plain --input_folder outputs/helloworld/fine --output_folder outputs/helloworld/frames
$BLENDER -noaudio --background --python generate.py -- --seed 0 --task render -g plain --input_folder outputs/helloworld/fine --output_folder outputs/helloworld/frames -p render.render_image_func=@flat/render_image 

in both i get white cubes not now ground creatures. Also for the video I noticed it not always outputs a scene but just leaves logs and it creates folder but they are empty. I am running 16Gb RAM but I have 32, maybe I should give it more than 1200 sec? Also, i am not sure but how can i expand the are of the "coarse area" in configs? As i noticed when i add fine it makes a nice big area around a tiny scene that is more detailed a mainly populated with tree outlines but only once i saw them as proper trunk. I also tried all the biomes this is just an example, but same results. Digging around today i saw you changed opengl_gt to blender_gt. BTW I haven't updated the files since the 9th of July. I hope maybe i just need to redownload worldgen. 

I hope this helps. 

Also attaching what i got with the forest biome as example.
<img width="1919" alt="compScreenshot 2023-07-22 143825" src="https://github.com/princeton-vl/infinigen/assets/97066252/3a9b0858-9641-452f-b9f6-fc6eba5924b0">
 
 Thanks a million!

**Comment by araistrick** [comment]
Sorry for the delay on this. It believe it is likely that creatures are present in the scene but just hidden from you - are there any creatures in the scene if you dig through the outliner in the top right? If so, you may need to follow the instructions in #94 to unhide them. Also ensure you are using the latest version and using fine/scene.blend not coarse/scene.blend.

**Comment by araistrick** [comment]
Closing due to inactivity
